net/acme-acmesh/files/hook.sh

```diff
@@ -45,6 +45,20 @@ get)
 
 			case $status in
 			0)
+				mkdir -p /etc/ssl/acme
+				if [ ! -e "/etc/ssl/acme/$main_domain.crt" ]; then
+					ln -s "$domain_dir/$main_domain.cer" "/etc/ssl/acme/$main_domain.crt"
+				fi
+				if [ ! -e "/etc/ssl/acme/$main_domain.key" ]; then
+					ln -s "$domain_dir/$main_domain.key" /etc/ssl/acme
```

Comment:
Let's add the filename to the target path here as well, so it's consistent with the other `ln` invocations.

Reply:
Done.